style.css

```diff
@@ -0,0 +1,127 @@
+@import url("https://fonts.googleapis.com/css2?family=Jost:ital,wght@0,100..900;1,100..900&display=swap");
+
+* {
+    font-family: "Jost", sans-serif;
+    margin: 0;
+    box-sizing: border-box;
+    background-color: #464a40;
+}
+
+h1 {
+    color: #e8ce3e;
+    margin: 20px;
+    padding-left: 40px;
+    font-size: 64px;
+}
+
+header {
+    display: flex;
+    justify-content: space-between;
+}
+
+.navbar {
+    display: flex;
+    justify-content: flex-start;
+}
+
+.filter-box,
+.sort-box,
+.random-box {
+    margin: 2rem;
+    padding-left: 30px;
+}
+
+.navbar-headers {
+    font-size: 22px;
+    font-weight: bold;
+    padding: 16px 16px 16px 0px;
+}
+
+.buttons {
+    background-color: #000000;
+    color: #e8ce3e;
+    padding: 8px 16px 8px 16px;
+    font-style: bold;
+    border-radius: 50px;
+    border: none;
+}
+
+.buttons:focus {
+    background-color: #cdcba6;
+    color: #000000;
+    outline: none;
+}
+
+.buttons:hover {
+    background-color: #cdcba6;
+    color: #000000;
+}
+
+#show-card-container {
+    display: grid;
+    grid-template-columns: 1;
+    gap: 20px;
+    margin: 1.5rem;
+    padding-left: 30px;
+    padding-right: 30px;
+}
+
+.card-img {
+    width: 100%;
+    height: 200px;
+    object-fit: cover;
+    border-radius: 12px;
+    margin-bottom: 1rem;
+}
+
+h2 {
+    font-size: 22px;
+    padding-bottom: 16px;
+}
+
+.card-info {
+    font-size: 18px;
+    font-weight: bold;
+    text-decoration: underline;
+}
+
+.description {
+    font-size: 16px;
+}
+
+.show-cards {
+    height: 100%;
+    padding: 16px 16px 24px 16px;
+    border-radius: 16px;
+}
+
+.show-cards:hover {
+    box-shadow: 0px 0px 30px 0px #3b3f36;
+    border: 7px solid #3b3f36;
+}
```

Comment:
When you apply an effect to the card, like in this case the big border, is making all the UI around move, and this is to be avoided. A solution would be to wrap the card in a div and give the div a fixed height 👍